### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Devadeut**
added helpful instruction to help the process go smoother while using Chillzone

# Description
Added keyboard shortcut for opening the Network tab

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,7 +102,7 @@ curl 'https://erp.iitkgp.ernet.in/Acad/timetable_track.jsp?action=second&dept=AE
 
 1. Login to the ERP
 2. Go to Academic -> Timetable -> Subject List with Timetable Slots
-3. Open the browser console. Switch to the Network tab
+3. Open the browser console. Switch to the Network tab [control + shift + j (cmd + shift + c in Mac)]
```
Comment (proffapt) [inline README.md:105]: Can write in more elaborated manner.
```suggestion
3. Open the browser console. Switch to the Network tab [Windows: `ctrl + shift + j` - macOS: `cmd + shift + c`]
```

**Comment by Devadeut** [comment]
Modified the previous commit.

**Comment by Devadeut** [comment]
@proffapt Kindly review and merge the PR.

**Review by proffapt — APPROVED** [review]
LGTM.

**Pull request merged**